load_eessi_extend_module.sh

```diff
@@ -98,9 +114,15 @@ else
         eessi_install_out=${TMPDIR}/eessi_install.out
         ok_msg="EESSI-extend/${EESSI_EXTEND_VERSION} installed, let's go!"
         fail_msg="Installing EESSI-extend/${EESSI_EXTEND_VERSION} failed, that's not good... (output: ${eessi_install_out})"
+
         # while always adding --try-amend=keep... may do no harm, we could make
         # an attempt to figure out if it is needed, e.g., when we are rebuilding
```

Comment:
@bedroge @ocaisa Since the `try-amend` was remove, we should also clean up the comment here, to avoid confusion going forward?